react-components/src/components/Architecture/pointsOfInterest/useSyncSelectedPoI.ts

```diff
@@ -0,0 +1,33 @@
+import { Dispatch, SetStateAction, useEffect, useMemo, useRef, useState } from 'react';
+import { PointOfInterest, PointsOfInterestDomainObject } from '../../../architecture';
+import { useOnUpdateDomainObject } from '../useOnUpdate';
+import { useRenderTarget } from '../../RevealCanvas';
+import { usePoIDomainObject } from './usePoIDomainObject';
+
+export const useSyncSelectedPoI = (
+  selectedPoi: PointOfInterest<any> | undefined,
+  setSelectedPoi: (poi: PointOfInterest<any> | undefined) => void
+) => {
```

Comment:
Can we get rid of the any here as well or it would demand to much effort?
Btw, these "any"s seems like a temporary definition as we are planning to publish this feature asap, isnt? I think it is fine to keep it for now but maybe we can think about it once we have more time.

Reply:
I didn't really plan for them to be temporary, no :sweat_smile: The reason I write `any`/`unknown` in these places, is that it just doesn't matter for the function which ID type the Point of Interest has